stable/rethinkdb/templates/rethinkdb-secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rethinkdb-password: {{ .Values.rethinkdbPassword | b64enc | quote }}
```

Comment:
I actually had it like that before, but there is an issue when you do that: https://github.com/kubernetes/charts/issues/980. I'm not really sure what would be the best way to combat that issue.